playgame.py

```diff
@@ -24,6 +25,17 @@ def main(argv):
     else:
         run_one_game(args)
 
+def create_dirs(path):
```

Comment:
minor nit: I would rather see this called `ensure_path()`. A few reasons:
1. the parameter is a path, so we would have consistent language there
2. create implies the path will always be made. The function checks if the path exists and does something if not
3. the function guarantees that a path exists, rather than makes it unconditionally (reword of 2)